log-cleanup/airflow-log-cleanup.py

```diff
@@ -97,7 +100,13 @@
 
 sleep ${WORKER_SLEEP_TIME}s
 
-MAX_LOG_AGE_IN_DAYS="{{dag_run.conf.maxLogAgeInDays}}"
+DAG_RUN_CONF="{{dag_run.conf}}"
+echo "DAG_RUN_CONF: $DAG_RUN_CONF"
+
+if [[ "$DAG_RUN_CONF" =~ .*"maxLogAgeInDays".* ]]; then
+  MAX_LOG_AGE_IN_DAYS="{{dag_run.conf.maxLogAgeInDays}}"
+fi
+
```

Comment:
@prakshalj0512  - this still fails for me when the var is not passed via run config because of a change in handling undefined variabled via jinja2.

https://airflow.apache.org/docs/apache-airflow/stable/upgrading-to-2.html#step-5-upgrade-airflow-dags

Reply:
Thanks for sharing this. I've updated the PR with the changes. Let me get it peer-reviewed and we can merge into master